exercises/csharp/TriviaGame/TriviaGame/Game.cs

```diff
@@ -135,16 +135,16 @@ private void AskQuestion()
 
         private string CurrentCategory()
         {
-            if (places[currentPlayer] == 0) return "Pop";
-            if (places[currentPlayer] == 4) return "Pop";
-            if (places[currentPlayer] == 8) return "Pop";
-            if (places[currentPlayer] == 1) return "Science";
-            if (places[currentPlayer] == 5) return "Science";
-            if (places[currentPlayer] == 9) return "Science";
-            if (places[currentPlayer] == 2) return "Sports";
-            if (places[currentPlayer] == 6) return "Sports";
-            if (places[currentPlayer] == 10) return "Sports";
-            return "Rock";
+            if (places[currentPlayer] == 0) return Category.Pop;
+            if (places[currentPlayer] == 4) return Category.Pop;
+            if (places[currentPlayer] == 8) return Category.Pop;
+            if (places[currentPlayer] == 1) return Category.Science;
+            if (places[currentPlayer] == 5) return Category.Science;
+            if (places[currentPlayer] == 9) return Category.Science;
+            if (places[currentPlayer] == 2) return Category.Sports;
+            if (places[currentPlayer] == 6) return Category.Sports;
+            if (places[currentPlayer] == 10) return Category.Sports;
```

Comment:
I like the way you have used properties on Category. Is there anyway you can group together the question numbers into one category? I struggled with this and tried comparing ints which obviously didn't work, I am going to look into other options.

Reply:
Yeah I hope we're going to go through the best way of refactoring this because I'm not happy with this solution either.